apps/products-feed/src/lib/editor-js-plaintext-renderer.ts

```diff
@@ -0,0 +1,87 @@
+interface ParagraphData {
+  text: string;
+}
+
+interface HeaderData {
+  text: string;
+  level: number;
+}
+
+interface ListData {
+  items: string[];
+  style: "unordered" | "ordered";
+}
+
+interface esjBlock {
+  id: string;
+  type: string;
+  data: ParagraphData | HeaderData | ListData | Record<never, never>;
+}
+
+interface ejsData {
+  version: string;
+  time: number;
+  blocks: esjBlock[];
+}
+
+const renderParagraph = (data: ParagraphData) => {
+  return data.text;
+};
+
+const renderHeader = (data: HeaderData) => {
+  return data.text + "\n";
+};
+
+const renderList = (data: ListData) => {
+  if (data.style === "ordered") {
+    return data.items.map((item, index) => `${index + 1}. ${item}`).join("\n");
+  }
+  return data.items.map((item) => `- ${item}`).join("\n");
+};
+
+const renderDelimiter = () => {
+  return "\n";
+};
+
+const renderBlock = (block: esjBlock) => {
+  switch (block.type) {
+    case "header":
+      return renderHeader(block.data as HeaderData);
+    case "paragraph":
+      return renderParagraph(block.data as ParagraphData);
+    case "list":
+      return renderList(block.data as ListData);
+    case "delimiter":
+      return renderDelimiter();
+    default:
+      return "";
+  }
+};
+
+const removeHtmlTags = (input: string) => {
+  /*
+   * The EditorJS used in the dashboard produces only a few one letter tags,
+   * like <b> or </s>, so we can use simpler regex to remove them
+   */
+  return input.replaceAll("&nbsp;", " ").replace(/<[^>]{1,2}>/g, "");
+};
```

Comment:
I think we could use separate tests for that function.

Reply:
Wanted to avoid exporting this function.
- there's test checking if style tags are removed
- added a new test to check if the description contains for example `<3` it's escaped as `&lt;3`

Are there other tests you think would improve the coverage?

Reply:
LGTM